src/components/Button.tsx

```diff
@@ -0,0 +1,17 @@
+import Link from "next/link";
+import React from "react";
+
+interface ButtonProps {
+  text: string;
+  link: string;
+}
+
+const Button = ({ text, link }: ButtonProps) => {
+  return (
+    <div className="rounded-md bg-blue-900 px-8 py-2 font-serif text-[vw] text-white">
```

Comment:
Instead of a div can we use the button component here. It's better for accessibility and makes more sense in this case.